server/models/Request.js

```diff
@@ -21,7 +21,11 @@ const requestSchema = new mongoose.Schema({
   },
   accepted: {
     type: Boolean,
-    default: null,
+    default: false,
+  },
+  declined: {
+    type: Boolean,
+    default: false,
```

Comment:
I'd prefer to use only `accepted` and make it `Boolean`. It will serve for both since `accepted: true` means declined.